src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -161,10 +161,12 @@ def get_privacy_notice(
     *,
     privacy_notice_id: str,
     db: Session = Depends(deps.get_db),
+    request: Request,
 ) -> PrivacyNotice:
     """
     Return a single PrivacyNotice
     """
+    unescape = request.headers.get("unescape-safestr")
```

Comment:
I'd like to do this at a higher level so that not every endpoint that wants to do this has to do it themselves. but I'm not familiar enough with fastapi to know where that kind of logic should go